packages/Takeover/src/components/LockBodyScroll.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
Can go into the helpers package for sure.

Reply:
Should it be in `helpers` dir or directly in `packages`? For example, we have `Guard` component in `packages`. And I think `LockBodyScroll` works on the same level of responsibility as `Guard` component.

Reply:
what about creating a new folder named `components`, https://github.com/acl-services/paprika/tree/master/packages/helpers/src ?